las_geoh5/import_files/params.py

```diff
@@ -0,0 +1,51 @@
+#  Copyright (c) 2024 Mira Geoscience Ltd.
+#
+#  This file is part of las-geoh5 package.
+#
+#  All rights reserved.
+#
+
+from pydantic import BaseModel, ConfigDict
+
+LAS_GEOH5_STANDARD = {
+    "well_name": "WELL",
+    "depth_name": "DEPTH",
+    "collar_x_name": "X",
+    "collar_y_name": "Y",
+    "collar_z_name": "ELEV",
+}
+
+
+class NameOptions(BaseModel):
+    """
+    Stores options for naming of dillhole parameters in las files.
+
+    :param depth_name: Name of the depth field.
+    :param collar_x_name: Name of the collar x field.
+    :param collar_y_name: Name of the collar y field.
+    :param collar_z_name: Name of the collar z field.
+    """
+
+    well_name: str = LAS_GEOH5_STANDARD["well_name"]
```

Comment:
Why not just assign the default string directly?

Reply:
Yep, I think I was using in more than one place before i introduced the pydantic class.. Removed.